mentorship ios/Service/Networking/ProfileAPI.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ProfileAPI.swift
+//  Created on 22/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+import Combine
+
+class ProfileAPI: ProfileService {
+    private var cancellable: AnyCancellable?
+    let currentProfile = ProfileViewModel().getProfile()
+    
+    func getProfile(completion: @escaping (ProfileModel.ProfileData) -> Void) {
```

Comment:
can we return  `-> AnyCancellable?` and remove local variable?

Reply:
We can, but then the Views will have to import Combine. How will it benefit btw?

Reply:
there is no benefit, you are right :)

Reply:
Just realised, there is (not in our case though). Cancellable can be used to explicitly cancel the network request from the view (we're not doing that though).

Reply:
you can provide cancel method to service which will do the job. It's ok to keep it here, less variables to sync on View layer.